fuzzing/fuzz_decoder_ppp_new.c

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2018 Code Intelligence. All rights reserved.
```

Comment:
What is the difference with the other one ? since the target function seems in both cases `DecodePPP`